ubuntu/standard/versions

```diff
@@ -1,2 +1,2 @@
-14.04 trusty 7
+14.04 trusty 8
```

Comment:
This updates 14.04 to use Java 8 as the default.

Reply:
Sounds good.